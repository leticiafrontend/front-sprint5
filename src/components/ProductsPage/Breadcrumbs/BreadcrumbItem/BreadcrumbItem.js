import React from 'react'
import { Item, Link } from './style'

export const BreadcrumbItem = ({ link, name }) => (
  <Item>
    <Link href={link}>{name}</Link>
  </Item>
)
