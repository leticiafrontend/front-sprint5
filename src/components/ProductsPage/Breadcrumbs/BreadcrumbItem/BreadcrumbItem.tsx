import React from 'react';
import { Item, Link } from './style';

interface PropsBreadcrumbsItem {
  link: string;
  name: string;
}

export const BreadcrumbItem: React.FC<PropsBreadcrumbsItem> = ({
  link,
  name,
}) => (
  <Item>
    <Link href={link}>{name}</Link>
  </Item>
);
