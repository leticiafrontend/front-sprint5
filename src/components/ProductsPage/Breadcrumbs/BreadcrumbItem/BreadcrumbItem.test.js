import React from 'react'
import { render } from '@testing-library/react'
import { BreadcrumbItem } from './BreadcrumbItem'

describe('Componente de item do breadcrumbs', () => {
  it('O snapshot do componente deve permanecer sempre o mesmo', () => {
    const { container } = render(<BreadcrumbItem link='/home' name='Home' />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
