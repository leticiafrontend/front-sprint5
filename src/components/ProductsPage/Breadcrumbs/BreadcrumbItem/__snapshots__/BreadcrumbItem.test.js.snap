// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente de item do breadcrumbs O snapshot do componente deve permanecer sempre o mesmo 1`] = `
<li
  class="sc-bdnxRM lbqNSD"
>
  <a
    class="sc-gtsrHT"
    href="/home"
  >
    Home
  </a>
</li>
`;
